portfolio/src/main/webapp/index.html

```diff
@@ -73,11 +76,9 @@ <h2 id="gettoknowme">Get To Know Me!</h2>
             <li>CONTACT INFO:</li>
             <li><hr style="height:5px;background-color:#d9d5fc;color:#d9d5fc;border-width:0;
             width:270px;margin-left:0px"></li>
-            <li><a href="#https://www.linkedin.com/in/sofia-fernandez-chacin-242787175/">LinkedIn</a>  <a 
-            href="#https://github.com/sofifch">GitHub</a>  <a 
+            <li><a href="https://www.linkedin.com/in/sofia-fernandez-chacin-242787175/">LinkedIn</a>  <a 
```

Comment:
All the links are in the same block yes? So the browser will line break them as it sees fit. You might want to so something with markup to make sure that you get what you want.

Reply:
I agree they are in the same block, seeing that they are in the same <li> element. However, I am not quite sure what you mean with markup and line breaking.